userguide/storage.rst

```diff
@@ -23,13 +23,12 @@ these options:
 * :ref:`Multipaths`: View multipath information for systems with
   compatible hardware.
 
-#ifdef truenas
-.. note:: When using an HA (High Availability) %brand% system,
+
+.. note:: When using a |enterprise| HA (High Availability) %brand% system,
```

Comment:
Isn't an article still needed here? If I'm undertsanding the substitution right, this would read:
`When using TrueNAS Enterprise HA (High Availablility) TrueNAS system,`
Do we need an `a` or do we need to s/ `systems`

Reply:
forgot to remove the extra %brand%, it should read:
"When using a TrueNAS Enterprise HA (High Availability) system,"